plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/StandardColumnMappings.java

```diff
@@ -532,9 +532,9 @@ public static LongTimestamp toLongTimestamp(TimestampType timestampType, LocalDa
     {
         long precision = timestampType.getPrecision();
         checkArgument(precision > TimestampType.MAX_SHORT_PRECISION, "Precision is out of range: %s", precision);
-        Instant instant = localDateTime.atZone(UTC).toInstant();
-        long epochMicros = instant.getEpochSecond() * MICROSECONDS_PER_SECOND + floorDiv(instant.getNano(), NANOSECONDS_PER_MICROSECOND);
-        int picosOfMicro = (instant.getNano() % NANOSECONDS_PER_MICROSECOND) * PICOSECONDS_PER_NANOSECOND;
+        long epochMicros = localDateTime.toEpochSecond(UTC) * MICROSECONDS_PER_SECOND
+                + floorDiv(localDateTime.getNano(), NANOSECONDS_PER_MICROSECOND);
+        int picosOfMicro = floorMod(localDateTime.getNano(), NANOSECONDS_PER_MICROSECOND) * PICOSECONDS_PER_NANOSECOND;
```

Comment:
floorDiv and floorMod seem redundant since nanos is a positive number

Reply:
Oh, good point, I'll fix that.